sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -615,6 +612,11 @@ def parts(self):
             import concurrent.futures
 
             def parse_responses(response):
+                from azure.core.pipeline import (
```

Comment:
Give the location of this file, you're not saving anything here, since to reach "azure.core.pipeline.transport" you would have imported anyway automatically "azure.core.pipeline" (sub-namespace), so I would suggest to not do it.